### DISCUSSION TITLE
how to create a nested bulleted list item in the template?

### DISCUSSION THREAD

**Discussion by kittenwhisky** [coddingtonbear/obsidian-web#191] [Q&A] [answered]
I'm appending the selected text as a nested bullet point. 
The template should first append a bullet point with the current page title as a hyperlink, then append the selected text as a nested bullet point underneath.

Here is my current template.

```
- [{{page.title}}]({{page.url}})
    {{#if page.selectedText}}
      - {{page.selectedText}}
    {{/if}}
```

The above template is not currently rendering the nested bullet point. The first bullet renders fine , but the nested bullet just shows as a dash not an actual bullet point - see screenshot below (2nd dash marked by yellow arrow).

![Screenshot 2024-04-28 164826](https://github.com/coddingtonbear/obsidian-web/assets/19669502/119981b8-00b3-440b-b38f-ced53cdfd42d)

What would be the syntax please for the nested bullet?

**Comment by 0xdiscas** [comment, marked as answer]
This is likely due to Obsidian's preview mode. It appears that only when there are 2 to 5 spaces, the nesting displays correctly during the preview.

If it’s not absolutely necessary, try removing any extra spaces.
```
- [{{page.title}}]({{page.url}})
{{#if page.selectedText}}
    - {{page.selectedText}}
{{/if}}
```
A classic solution is to enter the tab in an editor that does not convert tabs to spaces, copy it, and then replace the spaces used for indentation with this tab character.

For reference, the image below shows the changes made to the number of spaces.
![Obsidian_zdhUqL9s5i](https://github.com/coddingtonbear/obsidian-web/assets/168478596/25248d08-a4c3-49a9-b54b-d7fb41e5b995)

**Reply by kittenwhisky** [reply]
thanks so much!!!
